src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -955,7 +973,11 @@ private SafeProcessHandle GetProcessHandle(int access, bool throwIfExited)
                         }
                     }
                 }
-                return _processHandle;
+
+                // If we dispose of our contained handle we'll be in a bad state. NetFX dealt with this
```

Comment:
Is it possible to write a test to reproduce the problem that led you to fix this?